sceptre/cli/helpers.py

```diff
@@ -113,27 +113,34 @@ def _generate_json(stream):
 
 
 def _generate_yaml(stream):
-    if isinstance(stream, list):
+    kwargs = {
+        "default_flow_style": False,
+        "explicit_start": True
+    }
+
+    if type(stream) in [list, set]:
```

Comment:
The proper way to check for types is with `isinstance()`, as that will handle things like inheritance properly.

```suggestion
    if isinstance(stream, (list, set)):
```